src/routes/(site)/genuary/collage.svelte

```diff
@@ -0,0 +1,113 @@
+<script lang="ts">
+  import type { CollagePiece } from './collage';
+
+  export let data: CollagePiece[] = [];
+</script>
+
+<div class="grid-outer-wrapper">
+  <div class="grid-inner-wrapper">
+    {#each data as { src, view, alt: givenAlt }, i}
+      {@const alt = givenAlt || src}
+      {@const video = src.endsWith('.mp4') || src.endsWith('.mov')}
+      <details class="modal artwork-container">
+        <summary>
+          {#if video}
```

Comment:
It may be possible to simplify this if-statement using [`svelte:element`](https://svelte.dev/tutorial/svelte-element).

Reply:
Resolved by <https://github.com/EthanThatOneKid/acmcsuf.com/pull/761/commits/1121c0cc46742bc4ff3c653cef2372a7aa0a559b>.